content/articles/why-data-storage-is-shifting-to-the-blockchain/index.md

```diff
@@ -0,0 +1,60 @@
+## Why Data Storage is shifting to the blockchain
+
+Some years ago, the most embraced form of data storage was the traditional in-house data centers. But it was not too long before cloud storage started to replaced them. Cloud storage came with the promise of greater efficiency and more data storage capacity that attracted many businesses.
+
+In the same way, cloud storage is now facing competition from blockchain technology. Blockchain comes with the promise to solve data security challenges - an issue that cloud storage has found challenging to deal with exhaustively. With blockchain data storage, companies and individuals would enjoy the kind of data transparency not seen before. Thus, many companies are opting for this new technology with the hope that it could revolutionize data storage for the better.
+
+This article discusses blockchain as a more secure, cost-friendly alternative data storage system. It also gets leaders prepared of what to expect of blockchain as a data storage solution.
+
+### Blockchain storage vs. cloud storage
+When assessing cloud storage and comparing it to the blockchain, you realize immediate differences between the two systems. For a start, the cloud is a virtual space where you can access data remotely. You gain access to the cloud by surfing on the internet.
+
+Blockchain storage is a little dissimilar because it is an encrypted system that uses different methodologies to encrypt and store data in protected databases. The blockchain is built in such a way that the system keeps the data across several nodes that form a consensus on the exact position of the data.
+
```

Comment:
This section provides meaningful information. The comparison can be done by highlighting the key words in bold and explaining the differences that correspond to the key words. Here is an example:
- **Mutability:** In cloud storage, data is mutable, while in blockchain, data is immutable